vllm/config.py

```diff
@@ -86,6 +87,7 @@ def __init__(
         self.quantization = quantization
         self.enforce_eager = enforce_eager
         self.max_context_len_to_capture = max_context_len_to_capture
+        self.device = torch.device(device)
```

Comment:
I feel `device` is not a model's config but rather deserves its own config class like `DeviceConfig`. Could you make one?

Reply:
Done.